docs/source/development.rst

```diff
@@ -63,12 +63,29 @@ Creating a grammar
 `antlr <https://www.antlr.org>`_, a parser generator. ``antlr`` expects grammar files
 and produces parsing Python code.
 
+Additionally, code generated using ``speedy-antlr-tool`` package is used to parse SQL scripts
+in C++ for better performance. The parsed tree is then converted into the Python equivalent.
+
 If you want to adapt the grammar please have a look at ``pytsql/src/pytsql/grammar/tsql.g4``.
-All other files in ``pytsql/src/pytsql/grammar`` are generated by ``antlr``. You can
-generate these files by running
+All files in ``pytsql/src/pytsql/grammar/cpp_src/antlr4-cpp-runtime`` are taken directly from
```

Comment:
Have all the files from `github.com/antlr/antlr4/tree/master/runtime/Cpp/runtime/src` been copied or just a subset?
If a subset, how has this subset been determined?

Maybe a git permalink or a reference to a commit of the upstream code could be useful for the future.

Reply:
It was all files. I can add a link to the correct GitHub release.

Reply:
Done